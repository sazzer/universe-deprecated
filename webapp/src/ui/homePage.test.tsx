import { HomePage } from "./homePage";
import React from "react";
import { render } from "@testing-library/react";

test("It renders correctly", () => {
  const { container } = render(<HomePage />);
  expect(container).toMatchSnapshot();
});
