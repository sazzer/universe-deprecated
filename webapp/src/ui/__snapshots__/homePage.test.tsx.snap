// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It renders correctly 1`] = `
<div>
  <div
    class="row"
  >
    <div
      class="col-12 col-md-4 order-sm-1"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
    </div>
    <div
      class="col-12 col-md-8"
    >
      <h3>
        The Continent of Khorvaire
      </h3>
      <img
        alt="The Continent of Khorvaire"
        src="/landing.jpg"
        width="100%"
      />
    </div>
  </div>
</div>
`;
